tasks/main.yml

```diff
@@ -79,12 +82,51 @@
         ISO_path: "{{ file_path.stat.path if file_path.stat.exists else CD_path.stat.path }}"
 
     - name: Check if VBoxGuest additions ISO is mounted
-      shell: mount -l 2>/dev/null|awk '/VBOXADDITIONS/{print $3}'
+      shell: mount -l 2>/dev/null|awk '/VBox_GAs/{print $3}'
```

Comment:
This doesn't seem OS-independent. There should be a nicer way of checking whether it's mounted - the current function is bit clunky

Reply:
Ouch, that's not about OS. Looks like VirtualBox ISO volume name changed. And yes, mount detection fails for me a lot, too.

For both Debian and RPM distributions I have:

```
$ iso-info /usr/share/virtualbox/VBoxGuestAdditions.iso | grep '^Volume'
Volume      : VBox_GAs_6.1.18
```

Reply:
Thanks for that info, that helps. I'll try to find a cleaner way to detect a mounted disk: Maybe just probing for the file/binary?